### PR TITLE
COMMERCE-5017 Porting

### PR DESCRIPTION
**Pull request by riccardo-alberti**

### CODE REVIEW

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:reevaluate:1157401_4102

**Comment by riccardo-alberti** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-checkout-web/src/main/java/com/liferay/commerce/checkout/web/internal/util/PaymentProcessCommerceCheckoutStep.java [hunk 1]
```diff
@@ -101,10 +102,8 @@ public void render(
 				(HttpServletResponse)httpServletResponseWrapper.getResponse();
 		}
 
-		String paymentServletURL =
-			originalHttpServletResponse.encodeRedirectURL(
-				paymentProcessCheckoutStepDisplayContext.
-					getPaymentServletUrl());
+		String paymentServletURL = URLCodec.encodeURL(
+			paymentProcessCheckoutStepDisplayContext.getPaymentServletUrl());
```
Comment (riccardo-alberti) [inline modules/apps/commerce/commerce-checkout-web/src/main/java/com/liferay/commerce/checkout/web/internal/util/PaymentProcessCommerceCheckoutStep.java:106]: hey @topolik this change seems to create problems. 
image
Could you please check that this is the right place to encode the URL? Thx

Comment (topolik) [reply modules/apps/commerce/commerce-checkout-web/src/main/java/com/liferay/commerce/checkout/web/internal/util/PaymentProcessCommerceCheckoutStep.java:106]: Yeah, I think there should be no URL encoding at this place, i.e. neither 			```originalHttpServletResponse.encodeRedirectURL``` nor ```URLCodec.encodeURL``` Thanks!

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-checkout-web/src/main/java/com/liferay/commerce/checkout/web/internal/util/PaymentProcessCommerceCheckoutStep.java (hunk 1 above)
Comment (riccardo-alberti) [reply modules/apps/commerce/commerce-checkout-web/src/main/java/com/liferay/commerce/checkout/web/internal/util/PaymentProcessCommerceCheckoutStep.java:105]: @topolik thanks!

**Pull request closed without merging**